operator/bundle_helpers/patch-csv.py

```diff
@@ -202,6 +202,7 @@ def main():
               first_version=args.first_version,
               unreleased=args.unreleased,
               no_related_images=args.no_related_images,
+              inject_related_images=args.inject_related_images,
```

Comment:
There's an interdependency between `no-related-images` and `inject-related-images`. Namely, the combination `--no-related-images --inject-related-images` does not make sense and should lead to error. It does not cause any error currently. Rather than adding the error (which I would request if we stay with the current approach), I suggest we can do it a bit differently.

Since downstream does _not_ specify `--no-related-images`, we can redefine command-line arguments in a way that does not break downstream, and is not able to create a confusing combination.

Instead of two boolean flags I suggest using three-choice argument (see [choices](https://docs.python.org/3/library/argparse.html#choices)).

* `--related-images=downstream` (also default if omitted): call `rewrite(csv_doc, related_image_passthrough)` and delete `csv_doc['spec']['relatedImages']`.
* `--related-images=omit` - pass that instead of `--no-related-images` here <https://github.com/stackrox/stackrox/blob/b43432e39469cacd48d5da27086adc364b5e9537/operator/Makefile#L453-L453>: don't call `rewrite(...)` and delete `csv_doc['spec']['relatedImages']`.
* `--related-images=konflux` (or give other name if you like it): call `rewrite(...)` and populate `csv_doc['spec']['relatedImages']`.